Code Samples/BoxConsoleSample/README.md

```diff
@@ -0,0 +1,20 @@
+# Box Sample
+This sample is a simple C++ program that takes console input (box dimensions) and computes their sum (box volume).
+
+We use this example in our blog posts to illustrate new extension features.
+
+## Build and Debug Active File
+Available as of March 2019, "Build and Debug Active File" automatically configures the build tasks and kicks off a build and debug session.
+
+### Context Menu
+While editing a file in a workspace folder, you can right click in the editor field and select the "Build and Debug Active File" context menu option.
+
+![right click and select Build and Debug Active File](./Code Samples\BoxConsoleSample\build_debug_active_file.png)
```

Comment:
The image doesn't load.  I believe you can just put the filename here since it's in the same folder.